src/org/opensolaris/opengrok/web/Util.java

```diff
@@ -1294,7 +1294,7 @@ public static String linkify(String url, boolean newTab) {
                 return String.format("<a href=\"%s\"%s title=\"Link to %s\">%s</a>",
                         constructed.toString(),
                         newTab ? " target=\"_blank\"" : "",
-                        Util.htmlize(url),
+                        Util.encode(url),
```

Comment:
while you're at it, `encode()` should probably use `Appandable` like `htmlize()` and say that it actually appends the converted char.